llama-index-core/llama_index/core/postprocessor/node.py

```diff
@@ -283,6 +283,7 @@ class AutoPrevNextNodePostprocessor(BaseNodePostprocessor):
     infer_prev_next_tmpl: str = Field(default=DEFAULT_INFER_PREV_NEXT_TMPL)
     refine_prev_next_tmpl: str = Field(default=DEFAULT_REFINE_INFER_PREV_NEXT_TMPL)
     verbose: bool = Field(default=False)
+    response_mode: ResponseMode = Field(default=ResponseMode.REFINE)
```

Comment:
I just changed the default back, and also made the service context optional

Reply:
@nerdai @logan-markewich 
The reason I am changing the default response mode to refine instead of tree_summarize is because the subsequent code needs to check if the response contains the words "next", "previous", or "none". In fact, if the tree_summarize response mode is used, the LLM will use the summarize prompt, and its returned response does not meet the requirements of the following method _parse_prediction, which means it does not contain "previous", "next", or "none", resulting in an error.
![image](https://github.com/run-llama/llama_index/assets/104314597/2b13b633-2480-4f91-b25a-56d02220d0bf)

I have attached actual debug test screenshots to give you a more intuitive view of the issue.
![1](https://github.com/run-llama/llama_index/assets/104314597/d1ae4e29-b7af-4b22-b14c-cdb69ac7966f)

Reply:
I think this is just an issue for a specific case? You can still override the response mode in the constructor

The tree summarize prompt is nearly the same as the refine prompt fyi